hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRequest.java

```diff
@@ -1373,4 +1376,109 @@ protected void validateEncryptionKeyInfo(OmBucketInfo bucketInfo, KeyArgs keyArg
           keyArgs.getKeyName() + " in encrypted bucket " + keyArgs.getBucketName(), INVALID_REQUEST);
     }
   }
+
+  protected void addMissingParentsToCache(OmBucketInfo omBucketInfo,
+                                          List<OmDirectoryInfo> missingParentInfos,
+                                          OMMetadataManager omMetadataManager,
+                                          long volumeId,
+                                          long bucketId,
+                                          long transactionLogIndex) throws IOException {
```

Comment:
Please do not format method signature like this.  Whenever visibility / return type / method name / other modifiers are changed, we would have to reindent all parameters.

Reply:
https://github.com/apache/ozone/blob/master/hadoop-ozone/dev-support/intellij/ozone-style.xml

I'm not sure if it has anything about this particular preference.

Reply:
updated